tests/multiply.apln

```diff
@@ -0,0 +1,151 @@
+:Namespace multiply
+    Assert←#.unittest.Assert
+    stripToSameLen←#.utils.stripToSameLen
+    isDyalogClassic←#.utils.isClassic
+
+    complexMultiplyModel←{
+        (⎕FR≡1287) ∧ case≡'Hcmplx': ⍺×⍵ ⍝ Skipping cmplx for ⎕FR←1287
+        a c b d←∊(9 11○⊂)⍺ ⍵ 
+        ⍝ (a+bi)(c+di)=(ac−bd)+(ad+bc)i
+        x←((a model c)-(b model d))
+        y←((a model d)+(b model c))
+        x(⊣+¯11○⊢)y
+    }
+
+    model←{⍺{
+        0≡⍵:0
+        ⍝ is ⍵ or ⍺ complex
+        (1289≡⎕DR ⍵) ∨ (1289≡⎕DR ⍺): ⍺ complexMultiplyModel¨ ⍵
+        ⍺⌹¨÷⍵

```

Comment:
I would perhaps add a comment that this model is sometimes going to be a tiny bit imprecise, as it deals with floating point numbers. For example, multiplying two integers is always precise (as long as the result is integer), but this model computes the reciprocal, which might not be stored with full precision.